src/libcore/intrinsics.rs

```diff
@@ -520,6 +507,35 @@ extern "rust-intrinsic" {
     pub fn bswap32(x: u32) -> u32;
     /// Reverses the bytes in a `u64`.
     pub fn bswap64(x: u64) -> u64;
+}
+/// The following intrinsics are disallowed on PNaCl:
+/// We sneakily redirect them to libm or to a larger sized intrinsic.
```

Comment:
Fascinating! I think I'd like to take a look at this in the context of the rustc changes as well, but that should be fine if it's all lumped into #26148